Scripts/Web_Scrappers/CodeForces_Scraper/README.md

```diff
@@ -0,0 +1,18 @@
+# CodeForces Problem Scraper
+
+Takes A CodeForces Problem Code As A Command-Line Argument, Scrapes The Appropriate Problem, And Saves It To A Text File.
+
+## Pre-Requisites
+
+Run The Command 
+
+`pip install -r requirements.txt`
+
+## Instructions To Run
+
+Run The Command
+
+`python codeforces_scraper.py`
```

Comment:
Here the problem code must also be given

Reply:
Yes!